packages/web-forms/src/components/widgets/MultiselectDropdown.vue

```diff
@@ -87,8 +87,8 @@ if (props.question.appearances['no-buttons']) {
 
 	.p-multiselect-option {
 		&[aria-selected='true']::after {
-			content: '\e916';
-			font-family: 'owf-icomoon';
+			content: '\2713';
+			font-family: system-ui;
```

Comment:
This seems close enough to the check mark from Material Icon. It wasn't worth the effort to customize this component to use Material Icons (for now).